src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -66,8 +67,12 @@ public String getAuthToken() {
         return authToken;
     }
 
+    public boolean getBotUser() {
+        return botUser;
+    }
+
     public String getBuildServerUrl() {
-        if(buildServerUrl == null || buildServerUrl == "") {
+        if(buildServerUrl == null || "".equals(buildServerUrl)) {
```

Comment:
You may even be tempted to use `StringUtils.isBlank(buildServerUrl)` (https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isBlank(java.lang.String)) :-)